activerecord/lib/active_record/attributes.rb

```diff
@@ -131,22 +131,32 @@ module ClassMethods
       #       end
       #     end
       #   end
+      # #
```

Comment:
I think it was introduced by accident, did it? 🤔

Reply:
Yes -- good catch! fixed.